src/components/Character/Character.jsx

```diff
@@ -4,7 +4,10 @@ import {
 	CharactersContext as Characters,
 	ACTIONS
 } from "COMPONENTS/Characters/";
+import HeroImage from "COMPONENTS/HeroImage/";
 import { RENDER, s } from "./";
+import RENDER_TEXT from "SHARED/renderUtils.jsx";
```

Comment:
Not sure about this naming - `RENDER` works as namespace but `RENDER_TEXT` implies its a value rather than an interface - if you want to namespace like this it might be worth doing something like `RENDER.TEXT` instead, although that probably implies that your structure is too complex. I think we might as well just stick to `RENDER` here instead.

Reply:
Did this, as discussed:
 const RENDER = { ...RENDER_A, ...RENDER_B };

Works fine. Using A and B as i don't know of a naming convention that sidesteps the verb -> noun issue !

Reply:
Will do a CR tonight / tomorrow, but as for the verb -> noun issue, instead of `RENDER_TEXT`, you can just do `TEXT_RENDER` ;)